src/test/java/mil/dds/anet/test/resources/utils/GraphQlHelper.java

```diff
@@ -58,6 +58,14 @@ public GraphQlHelper(Client client, int localPort) {
     return graphQlClient.doGraphQlQuery(user, q, null, null, responseType);
   }
 
+  /**
+   * @return the requested object of any type
+   */
+  public Object getObject(Person user, String getQuery, Map<String, Object> variables) {
+    return graphQlClient.doGraphQlQuery(user, getQuery, variables,
+        new TypeReference<GraphQlResponse<Object>>() {});
+  }
```

Comment:
You could change this to:
```
  /**
   * @return the requested object of the given type
   */
  public <T extends Object> T getObjectOfType(Person user, String query,
      TypeReference<GraphQlResponse<T>> responseType) {
    return graphQlClient.doGraphQlQuery(user, query, null, responseType);
  }
```
to have type-safe calls (see above for what the calls would change to).